proto/prysm/v1alpha1/attestation.go

```diff
@@ -103,6 +108,14 @@ func (a *Attestation) Version() int {
 	return version.Phase0
 }
 
+// IsNil --
+func (a *Attestation) IsNil() bool {
+	if a == nil || a.Data == nil {
```

Comment:
Sorry if I am being a little dense.
If we are checking `if att == nil || att.IsNil() {` for example do we need to check `a == nil` here?

Reply:
that's because this one is a concrete type *Attestation, it's safe to just directly check the values here